gh-pr-context.py

```
@@ -88,16 +88,79 @@ def gh_api_jq(endpoint, jq_filter):
     return result.stdout.strip().replace("\r", ""), True
 
 
+def _setup_git_env():
```

Ruff (0.15.15)

[warning] Missing return type annotation for private function `_setup_git_env`

Add return type annotation: `None`

(ANN202)